src/ui/units/editor/constants/templates/d3-chart.ts

```diff
@@ -0,0 +1,42 @@
+import {EDITOR_TYPE, EditorTemplatesQA} from '../../../../../shared/constants';
+
+export default {
+    qa: EditorTemplatesQA.D3,
+    name: 'd3',
+    type: EDITOR_TYPE.D3_NODE,
+    data: {
+        js: `module.exports = {
+    chart: {
+        margin: {
+            left: 10,
+            right: 10,
+            top: 10,
+            bottom: 10,
+        }
+    },
+    series: {
+        data: [
+            {
+                type: "bar-x",
+                name: "Fruits",
+                data: [
+                    { x: "Apples", y: 3 },
```

Comment:
From a performance point of view, it is the category index that should be stored in x, not the value itself.
Maybe it should be changed in this example?